spec/series-chart-spec.js

```diff
@@ -133,4 +133,50 @@ describe('dc.seriesChart', function() {
             expect(dots[0].length).toEqual(6);
         });
     });
+
+    describe('legends with hidable stacks', function () {
```

Comment:
@gordonwoodhull Added specs. I copied these from `legend-spec.js` and modified slightly to work with `seriesChart`. Would it be better to consolidate these in `legend-spec.js`? If so, I'd need to duplicate the `seriesChart` setup. Not a big deal, just let me know which approach you prefer.